### ISSUE TITLE
Can't overwrite req.params values

### ISSUE THREAD

**Issue by Shanison** [balderdashy/sails#1977]
I am using MongoDB V2.4.9 and Sails.js 0.10.0-rc8. I need to add some default sorting to a model's blueprint find methods. The way to do it with Sails V0.10 is to create a policy. Below is a very simple code:

```
module.exports = function(req, res, next) {
  var params = req.params.all();
  var sort = params.sort;

  // If the sort is not added, then we add the default sort
  if (!sort) {
     // both params.sort and req.params.sort doesn't work here
    req.params.sort = "createdAt desc";
  }

  return next();
}
```

And this policy is applied to find method in policies.js. However, the default sorting is not applied. I tried to debug by creating a find method in the controller, when I print out req.params again, the sort value is not undefined. 

I found this discussion for express.js http://stackoverflow.com/questions/18801661/modify-node-js-req-object-parameters

I am doing the same as indicated. Sails.js is considering one of the middleware, so it should work as well.  Does sails.js requires a different way to do it? 

One work around is to use is to use req.options.sort as when parsing sort, sails also will consider req.options.sort based on this code https://github.com/balderdashy/sails/blob/master/lib/hooks/blueprints/actionUtil.js
```
req.options.sort = "createdAt desc";
```

**Comment by sailsbot** [comment]
Thanks for posting, @Shanison.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
